gapic-generator/templates/default/service/client/_self_configure_retry_policy.erb

```diff
@@ -0,0 +1,15 @@
+<%- assert_locals retry_policy -%>
+{
+  <%-if retry_policy.initial_delay_seconds -%>
+  initial_delay: <%= retry_policy.initial_delay_seconds =%>,
```

Comment:
Don't end with `=`. i.e. `%>` rather than `=%>`. Same in a few more cases below.